apps/design-system/src/components/OperationIcon.stories.tsx

```diff
@@ -0,0 +1,10 @@
+import { OperationIcon } from 'ui';
+
+export default {
+  title: 'OperationIcon',
```

Comment:
Remove this line. It's causing the OperationIcon component to appear on the top level of the design system instead of inside components.